tests/baselines/reference/internalAliasClassInsideTopLevelModuleWithoutExport.js

```diff
@@ -32,10 +32,10 @@ var cReturnVal = exports.cProp.foo(10);
 
 
 //// [internalAliasClassInsideTopLevelModuleWithoutExport.d.ts]
+import xc = x.c;

```

Comment:
Another case of odd reordering, especially since if the source were written this way it would have failed at runtime.